src/api/status/public/js/build-log/build-header.js

```diff
@@ -0,0 +1,43 @@
+const resMsg = document.getElementById('response-msg');
+const buildHeaderInfo = document.getElementById('build-header-info');
+const buildSender = document.getElementById('build-sender');
+const buildSenderName = document.getElementById('build-sender-name');
+const buildSenderImg = document.getElementById('build-sender-img');
+const buildGitSHA = document.getElementById('build-git-sha');
+const buildResult = document.getElementById('build-result');
+const buildStarted = document.getElementById('build-started');
+const buildDuration = document.getElementById('build-duration');
+
+function renderBuildInfo({ githubData, startedAt, stoppedAt, result }) {
+  resMsg.toggleAttribute('hidden');
+  buildHeaderInfo.toggleAttribute('hidden');
```

Comment:
This makes the build header switch between loading state and info state every 5s. Like, 5s of info and 5s of loading.

Reply:
You have any suggestions on how to fix it? I've tried using `setAttribute` and `removeAttribute` here but it doesn't seems to do the trick

Reply:
Maybe the function should only render the build info, not toggling between two states (i.e. only render info state)? I don't think there's a case where toggling between those two is necessary.